web/app.js

```diff
@@ -188,6 +189,12 @@ const PDFViewerApplication = {
     } catch (ex) {
       console.error(`initialize: "${ex.message}".`);
     }
+    if (typeof PDFJSDev !== "undefined" && PDFJSDev.test("MOZCENTRAL")) {
+      // We want to load the image-to-text AI engine as soon as possible.
+      this.mlManager = new MLManager({
+        enableAltText: AppOptions.get("enableAltText"),
+      });
+    }
     if (AppOptions.get("pdfBugEnabled")) {
       await this._parseHashParams();
     }
```

Comment:
Please flip the order of these blocks, since generally speaking *nothing else* should be initialized until the `AppOptions` are fully set. Besides, in practice this won't delay the `mlManager`-init since "normal" users don't have debugging enabled.
```suggestion
    if (AppOptions.get("pdfBugEnabled")) {
      await this._parseHashParams();
    }
    
    if (typeof PDFJSDev !== "undefined" && PDFJSDev.test("MOZCENTRAL")) {
      // We want to load the image-to-text AI engine as soon as possible.
      this.mlManager = new MLManager({
        enableAltText: AppOptions.get("enableAltText"),
      });
    }
```